catalog/app/containers/Bucket/PackageDialog/MetaInput.tsx

```diff
@@ -246,7 +257,8 @@ export const MetaInput = React.forwardRef<HTMLDivElement, MetaInputProps>(
     const [locked, setLocked] = React.useState(false)
 
     // used to force json editor re-initialization
-    const [jsonEditorKey, setJsonEditorKey] = React.useState(1)
+    const [jsonInlineEditorKey, setJsonInlineEditorKey] = React.useState(1)
```

Comment:
do we really need two different keys here? seems like they are updated simultaneously anyways

Reply:
There are two JsonEditors and they update each other respectively with different react keys.
I can't use one key: inline JsonEditor shouldn't re-render on inline change, because otherwise it loses focus

Reply:
Maybe, I'll add one layer of abstraction (one more component) with value/onChange interface, and remove these keys